Makefile

```diff
@@ -93,24 +62,15 @@ gen-files .generate_files: lint-cache-dir .generate_execs clean-generated
 
 	# generate openapi
 	$(DOCKER_RUN) $(CALICO_BUILD) \
-	   sh -c '$(GIT_CONFIG_SSH) $(BINDIR)/openapi-gen \
+	   sh -c '$(GIT_CONFIG_SSH) openapi-gen \
 		--v 1 --logtostderr \
 		--go-header-file "/go/src/$(PACKAGE_NAME)/hack/boilerplate/boilerplate.go.txt" \
 		--input-dirs "$(PACKAGE_NAME)/pkg/apis/projectcalico/v3,k8s.io/api/core/v1,k8s.io/api/networking/v1,k8s.io/apimachinery/pkg/apis/meta/v1,k8s.io/apimachinery/pkg/version,k8s.io/apimachinery/pkg/runtime,k8s.io/apimachinery/pkg/util/intstr,$(PACKAGE_NAME)/pkg/lib/numorstring" \
 		--output-package "$(PACKAGE_NAME)/pkg/openapi"'
-	$(DOCKER_GO_BUILD) \
```

Comment:
I'm not sure why this was here - all of the contents of the generated file were duplicates of functions already in the main openapi generation, so I think it's safe to remove? Could be wrong, though.

Leaving it was causing problems as for some reason it was deciding to use the wrong imports, unlike the main openapi file which was constructed correctly.

Reply:
No sure why it is in the Makefile in the first place. Yeah, we should just remove this.